api/static/rest_framework_swagger/css/osf_screen.css

```diff
@@ -0,0 +1,1232 @@
+/* http://meyerweb.com/eric/tools/css/reset/ v2.0 | 20110126 */
+.swagger-ui-wrap html,
```

Comment:
The static files are bundled within the rest_framework_swagger distribution. Running `manage.py collectstatic`  puts them in this static folder, but this is a *build* step. These assets are versioned by rest_framework_swagger